core/chains/evm/client/chain_client.go

```diff
@@ -0,0 +1,229 @@
+	ctx context.Context,
+	attempt interface{},
+	blockNumber *big.Int,
+) (rpcErr []byte, extractErr error) {
+	return c.multiNodeClient.SelectNode().RPCClient().CallContract(ctx, attempt, blockNumber)
+}
+
+// ChainID makes a direct RPC call. In most cases it should be better to use the configured chain id instead by
+// calling ConfiguredChainID.
+func (c *ChainClient) ChainID(ctx context.Context) (*big.Int, error) {
+	return c.multiNodeClient.SelectNode().RPCClient().ChainID(ctx)
+}
+
+func (c *ChainClient) CodeAt(ctx context.Context, account common.Address, blockNumber *big.Int) ([]byte, error) {
+	return c.multiNodeClient.SelectNode().RPCClient().CodeAt(ctx, account, blockNumber)
+}
+
+func (c *ChainClient) ConfiguredChainID() *big.Int {
+	return c.multiNodeClient.SelectNode().ConfiguredChainID()
+}
+
+func (c *ChainClient) EstimateGas(ctx context.Context, call any) (gas uint64, err error) {
+	return c.multiNodeClient.SelectNode().RPCClient().EstimateGas(ctx, call)
+}
+
+func (c *ChainClient) FilterEvents(ctx context.Context, query ethereum.FilterQuery) ([]types.Log, error) {
+	return c.multiNodeClient.SelectNode().RPCClient().FilterEvents(ctx, query)
+}
+
+func (c *ChainClient) IsL2() bool {
+	return c.ChainType().IsL2()
+}
+
+func (c *ChainClient) LatestBlockHeight(ctx context.Context) (*big.Int, error) {
+	return c.multiNodeClient.SelectNode().RPCClient().LatestBlockHeight(ctx)
+}
+
+func (c *ChainClient) LINKBalance(ctx context.Context, accountAddress common.Address, linkAddress common.Address) (*assets.Link, error) {
+	return c.multiNodeClient.SelectNode().RPCClient().LINKBalance(ctx, accountAddress, linkAddress)
+}
+
+func (c *ChainClient) PendingSequenceAt(ctx context.Context, addr common.Address) (evmtypes.Nonce, error) {
+	return c.multiNodeClient.SelectNode().RPCClient().PendingSequenceAt(ctx, addr)
+}
+
+func (c *ChainClient) SendEmptyTransaction(
+	ctx context.Context,
+	newTxAttempt func(seq evmtypes.Nonce, feeLimit uint32, fee *assets.Wei, fromAddress common.Address) (attempt any, err error),
+	seq evmtypes.Nonce,
+	gasLimit uint32,
+	fee *assets.Wei,
+	fromAddress common.Address,
+) (txhash string, err error) {
+	return c.multiNodeClient.SelectNode().RPCClient().SendEmptyTransaction(ctx, newTxAttempt, seq, gasLimit, fee, fromAddress)
+}
+
+func (c *ChainClient) SendTransaction(ctx context.Context, tx *types.Transaction) error {
+	main := c.multiNodeClient.SelectNode()
+	for _, n := range c.multiNodeClient.NodesAsSendOnlys() {
+		if n == main {
+			// main node is used at the end for the return value
+			continue
+		}
+		// Parallel send to all other nodes with ignored return value
+		// Async - we do not want to block the main thread with secondary nodes
+		// in case they are unreliable/slow.
+		// It is purely a "best effort" send.
+		// Resource is not unbounded because the default context has a timeout.
+		c.multiNodeClient.WrapSendOnlyTransaction(ctx, c.logger, tx, n, sendOnlyTransaction)
```

Comment:
If we don't implement SendTransaction inside MutliNodeClient we end up with this awkward implementation. It might make sense to include SendTransaction, although it means we will also have to include the `NewSendOnlyErrorReturnCode` method.